src/test/java/org/jacp/repository/QuestionRepositoryTest.java

```diff
@@ -1,50 +1,36 @@
 package org.jacp.repository;
 
+import org.jacp.AbstractIntegrationTestBase;
 import org.jacp.entity.QuestionEntity;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.junit.jupiter.MockitoExtension;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlGroup;
+
 import java.util.Optional;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
- * @author saffchen created on 11.08.2023
+ * @author saffchen created on 22.08.2023
  */
-@ExtendWith(MockitoExtension.class)
-class QuestionRepositoryTest {
-    @Mock
+@SqlGroup({
+        @Sql("/sql/dropData.sql"),
+        @Sql("/sql/data.sql")
```

Comment:
Minor: because there will be a lot of files it's better to rename those files, making them more related to implemented feature (adding unique trello-id, adding some functionality keywords and so on).

Reply:
Fixed